app/assets/javascripts/jquery-mentions-input/jquery.mentionsInput.js

```diff
@@ -470,7 +470,7 @@
     //Resets the text area
     function resetInput(currentVal) {
       mentionsCollection = [];
-      var mentionText = utils.htmlEncode(currentVal);
+      var mentionText = currentVal;
```

Comment:
This plugin is only ever used in the buddy-request view, and this was converting `<!here>` to `&lt;!here&gt;` after document-ready, which is what was breaking the `@here` mention in Slack. The content being filtered here is just going into a textarea, so it should be okay to allow "raw" html tags in there. If nobody is using this mention auto-complete plugin, we could just remove the whole thing.